bindings/generate_bindings_py.py

```diff
@@ -324,7 +324,10 @@ def gen_class(klass: swagger_parser.Class) -> Code:
 
 def gen_enum(enum: swagger_parser.Enum) -> Code:
     out = [f"class {enum.name}(enum.Enum):"]
-    out += [f'    {v} = "{v}"' for v in enum.members]
+    prefix = os.path.commonprefix(enum.members)
+    if not prefix.endswith("_"):
+        prefix = ""
+    out += [f'    {v[len(prefix):]} = "{v}"' for v in enum.members]
```

Comment:
Nit: I think that it's _really_ hard to read anything even kinda involved inside an fstring.

Can you rewrite this to do the `len(prefix)` outside the fstring?

```python
skip = len(prefix) if prefix.endswith("_") else 0
out += [f' {v[skip:]} = "{v}"' for v in enum.members]
```